test/token/ERC721/extensions/ERC721Royalty.test.js

```diff
@@ -0,0 +1,192 @@
+const { BN, constants, expectRevert } = require('@openzeppelin/test-helpers');
+const { expect } = require('chai');
+const { ZERO_ADDRESS } = constants;
+
+const { shouldSupportInterfaces } = require('../../../utils/introspection/SupportsInterface.behavior');
+
+const ERC721RoyaltyMock = artifacts.require('ERC721RoyaltyMock');
+
+contract('ERC721Royalty', function (accounts) {
+  const [ account1, account2 ] = accounts;
+  const tokenId1 = new BN('1');
+  const tokenId2 = new BN('2');
+  const salePrice = new BN('1000');
+  const royaltyFraction = new BN('10');
+
+  beforeEach(async function () {
+    this.token = await ERC721RoyaltyMock.new('My Token', 'TKN');
+
+    await this.token.mint(account1, tokenId1);
+    await this.token.mint(account1, tokenId2);
+  });
+
+  it('calls supports interface', async function () {
+    const result = await this.token.supportsInterface('0x2a55205a');
+    const expected = true;
+
+    expect(result).to.not.equal(undefined);
+    expect(result).to.equal(expected);
+  });
+
```

Comment:
Without this it doesnt pass the coverge test, it doesnt recognize it being tests on the `shouldSupportInterfaces ` method

Reply:
In `shouldSupportIntrefaces` we skip this check in coverage because it can fail due to gas limit. (Coverage instrumentation increases gas cost and ERC165 has a 30k gas requirement.)

So it is acceptable if coverage doesn't recognize it.

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/d95cdaae8e26b3e479ca2ad70cc7c5a303444392/test/utils/introspection/SupportsInterface.behavior.js#L103-L104

Reply:
commit :3d826f8f